federation-js/src/service/printSupergraphSdl.ts

```diff
@@ -246,10 +247,14 @@ function printTypeJoinDirectives(
 
   // We don't want to print an owner for interface types
   const shouldPrintOwner = isObjectType(type);
+
+  const enumValue = context.graphNameToEnumValueName?.[ownerService];
```

Comment:
This looks good, but there are three other lookups in `context.graphNameToEnumValueName` in this file that use the same pattern that are worth fixing too?

Reply:
a1156a85